docs/kratos/social-signin/55_discord.mdx

```diff
@@ -4,8 +4,66 @@ title: Add Discord as a social sign-in provider in Ory
 sidebar_label: Discord
 ---
 
+```mdx-code-block
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+```
+
 # Discord
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add Discord as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the Discord logo to start the configuration.
+3. Copy the **Redirect URI** and save it for later use.
+4. [Create a Discord OAuth2 application](https://discord.com/developers/applications) by clicking the **New Application** button.
+5. After creating the application, navigate to the **OAuth2 | General** section of the registered application in Discord
+   and copy the following data to the corresponding fields in the form on the Ory Console:
+    - **CLIENT ID**
+    - **CLIENT SECRET**
+6. From the **OAuth2 | General** section in Discord,
```

Comment:
Is there an UI element labeled exactly like this (using the vertical divider)?

Reply:
"OAuth2 | General" is an item that appears in the left-hand navigation pane. What convention should we using to indicate drilling down to a menu item? In the past, I have used the vertical bar to indicate this. Some docs teams use "->". What's the Ory convention?

Reply:
Here's an example of how I've done it so far: 

<img width="936" alt="image" src="https://user-images.githubusercontent.com/29116492/214579889-f8f6ae9c-da24-40bb-9182-66b81aceb3eb.png">